### PR TITLE
added the multi brain docs

### PR DESCRIPTION
**Pull request by xiaomaogy**

### CODE REVIEW

File: docs/Learning-Environment-Design-Brains.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Brains
 
-The Brain encapsulates the decision making process. Brain objects must be children of the Academy in the Unity scene hierarchy. Every Agent must be assigned a Brain, but you can use the same Brain with more than one Agent. 
+The Brain encapsulates the decision making process. Brain objects must be children of the Academy in the Unity scene hierarchy. Every Agent must be assigned a Brain, but you can use the same Brain with more than one Agent. You can also create several Brains, attach each of the Brain to one or more than one Agent.
```
Comment (vincentpierre) [inline docs/Learning-Environment-Design-Brains.md]: The fact that we can have multiple brains in the scene is not new. What is new is that now we can have multiple brains set to `External` in the scene and perform training in parallel.

Comment (xiaomaogy) [reply docs/Learning-Environment-Design-Brains.md]: @vincentpierre This is a good point. I've just added another sentence to address this.

**Comment by xiaomaogy** [comment]
@vincentpierre May I merge this now?

File: docs/Learning-Environment-Design-Brains.md (hunk 1 above)
Comment (RossMelbourne) [reply docs/Learning-Environment-Design-Brains.md]: I am attempting imitation learning with multiple brains set to external.  Basically two sets of Teacher/Student agents each with a brain.   By default only one bytes file is exported at the end of the learn.py process (after pressing ctrl-c).  How do I get two bytes files for the two brains I am training please?

**Pull request merged**